addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -838,6 +838,11 @@ body.editor_enable.editor_has_snippets {
             }
         }
 
+        // Buttons with an icon but no text, so no possible overflow
```

Comment:
@younn-o will probably put that diff in https://github.com/odoo/odoo/pull/76313 too so his button are fixed 
![image](https://user-images.githubusercontent.com/30048408/133771770-bd2b2648-5308-4435-acf9-19b6cbcc1b63.png)